TTHAnalysis/python/plotter/ttH-multilepton/mca-includes/mca-2lss-sigprompt.txt

```diff
@@ -9,7 +9,7 @@ TTW     : TTWToLNu_fxfx+TTWToLNu_fxfx_PS : xsec : LepGood1_isMatchRightCharge &&
 TTZ     : TTZToLLNuNu_amc : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ;  FillColor=ROOT.kSpring+2, Label="ttZ ", years="2016\,2018"
 TTZ     : TTZToLLNuNu_amc+TTZToLLNuNu_amc_PS : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ;  FillColor=ROOT.kSpring+2, Label="ttZ   ", years="2017"
 TTZ     : TTZToLLNuNu_m1to10 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ;  FillColor=ROOT.kSpring+2
-TTZ	: TTJets_DiLepton : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kSpring+2
+TTZ_test	: TTJets_DiLepton : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kSpring+2 #added this for mcdumping test
```

Comment:
If it's just a test, just call it test_ttz and leave the original TTZ line as it is

Reply:
DONE